config/webpack.config.js

```diff
@@ -97,6 +97,10 @@ module.exports = env => {
     ENVIRONMENTS.VAGOVPROD,
   ].includes(buildOptions.buildtype);
 
+  const useHashFilenames = [ENVIRONMENTS.VAGOVPROD].includes(
+    buildOptions.buildtype,
+  );
```

Comment:
So we can still optimize staging assets, this update is just isolated to testing file hashes and making sure nothing breaks